Dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM alpine:3.15
+FROM alpine:3.15.4
```

Comment:
@snvector Is there a specific reason why you did pin this version? We currently only update minor versions.